build_tools/docker/cmake-bazel-frontends/Dockerfile

```diff
@@ -20,7 +20,7 @@ COPY --from=gcr.io/iree-oss/bazel@sha256:5e52c7d43b6fdff35d884b8b8b92b1b6e2151d6
   /usr/bin/
 
 # Install tensorflow.
-RUN python3 -m pip install tf-nightly==2.6.0.dev20210527
+RUN python3 -m pip install tf-nightly==2.7.0.dev20210726
```

Comment:
errors on https://source.cloud.google.com/results/invocations/bcfe13d4-ea7c-477b-a4ab-e7b442501893/targets/iree%2Fgcp_ubuntu%2Fcmake-bazel%2Flinux%2Fx86-turing%2Fgoogle/log should be fixed, but might need a different version of tensorflow?

seems like something is assuming CUDA is present:
```
2021-07-29 02:39:13.503152: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1829] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.

Traceback (most recent call last):
  File "/tmpfs/src/github/iree/integrations/tensorflow/e2e/batch_to_space_nd_test.py", line 49, in <module>
    app.run(main)
  File "/usr/local/lib/python3.6/dist-packages/absl/app.py", line 301, in run
    callback()
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/distribute/combinations.py", line 458, in _set_total_phsyical_gpus
    context.context().list_physical_devices("GPU"))
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/eager/context.py", line 1356, in list_physical_devices
    self._initialize_physical_devices()
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/eager/context.py", line 1320, in _initialize_physical_devices
    devs = pywrap_tfe.TF_ListPhysicalDevices()
tensorflow.python.framework.errors_impl.InternalError: Cannot dlopen all CUDA libraries.
```

normally that's just a warning, but it might be crashing some tests now?

(many parts of that really bug me...)

Reply:
I can try updating again. I just wish there was a better way to validate this.